cpp/density/SphereVoxelization.cc

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) 2010-2019 The Regents of the University of Michigan
+// This file is from the freud project, released under the BSD 3-Clause License.
+
+#include <cmath>
+#include <stdexcept>
+#include <tbb/tbb.h>
```

Comment:
Need to replace with specific tbb headers (see #605).

Reply:
There is no explicit tbb in this file, so if I understand correctly, these need to be the tbb headers required for forLoopWrapper, right?